modules/zstd/1.5.5/patches/add_build_file.patch

```diff
@@ -0,0 +1,31 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,28 @@
++""" Builds zstd.
++"""
++
++load("@rules_foreign_cc//foreign_cc:defs.bzl", "cmake")
++
++filegroup(
++    name = "all_srcs",
++    # TODO(mvukov) Minimize srcs.
++    srcs = glob(["**"]),
++)
++
++cmake(
++    name = "zstd",
++    build_args = [
++        "--",
++        "-j4",
```

Comment:
This can be problematic as Bazel won't know that this action uses four CPU cores and may overschedule, which can result in test timeouts. Do you know how much of a difference this flag makes in terms of build time?

Reply:
I completely re-wrote the BUILD file in pure Bazel, also added tests.